packages/discord-snippet/0.1.0/README.md

```diff
@@ -0,0 +1,40 @@
+# Espanso GIT
```

Comment:
Do you mean to leave "Espanso GIT" this as the heading?

Reply:
Ah, i didn't see that, i'll change that